### ISSUE TITLE
Attachments don't load on dedicated server

### ISSUE THREAD

**Issue by keli5** [HaodongMo/ArcCW#155]
On my dedicated server, during startup, I get a million lines of this:
`!!!! Attachment <x>.lua has errors! lua/arccw/shared/sh_loadatts.lua:36: attempt to index a nil value`
One for seemingly every attachment. Ingame, no attachments spawn/are visible when applied to the gun, including default ones like the stock. I'm using the entire ArcCW collection from Steam. The server addon collection is here: https://steamcommunity.com/sharedfiles/filedetails/?id=3240260960

The line that contains the error: 
https://github.com/HaodongMo/ArcCW/blob/a268cde1a6f9e1a99e702d7524ace91fdf37609b/lua/arccw/shared/sh_loadatts.lua#L36

I also have these as filesystem addons:
https://github.com/LittleBigBug/chat-cosmetics
https://github.com/hoksalot/gm-psi
https://github.com/ZionDevelopers/sui-scoreboard

**Comment by keli5** [comment]
Update: After some testing, this seems to happen without the filesystem addons. This issue also doesn't occur on a local, multiplayer listen server with only the same addon collection installed

**Comment by Fesiug** [comment]
It's super strange and unknown how or why this happens, but there's a convar that allows you to disable generating of these attachment entities (arccw_reloadatts_registerentities) as a workaround.
